stories/bees.stories.tsx

```diff
@@ -30,18 +30,25 @@ interface StoreState extends UndoState {
 
```

Comment:
The interface above this line is missing `ignored: number;`. That is my issue from the past but got errors when I ran the storybook. Could you fix that? Thanks!